fhir-server/pom.xml

```diff
@@ -168,6 +168,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.ibm.cos</groupId>
+            <artifactId>ibm-cos-java-sdk</artifactId>
```

Comment:
I think we talked about potentially going straight to the aws client (removing the ibm-cos dependency), but agree it makes more sense to stick with the ibm-cos to match what we currently have in bulkdata-webapp.
@punktilious heads up in case it conflicts with any changes you made on the s3 side.

Reply:
I also considered whether its worth trying to make this a shared dependency between fhir-server-webapp and bulkdata-webapp, but I don't think its worth the pain.  Better to just package it with each war.